Maximilian _udemy/functions/Notes.md

```diff
@@ -15,4 +15,10 @@ If we store a function in an object, then the function is said as "method"(a fun
 | Hoisted to top,can be declared anywhere in the file(also after its use)  |hoisted to top but cannot initialized/defined(can't be declared after its use)   |
 
 
```

Comment:
consolidate all notes under one readme instead of keeping these in with src files

Reply:
done